client/src/index.css

```diff
@@ -2,18 +2,27 @@
 @tailwind components;
 @tailwind utilities;
 
+html {
+  scrollbar-gutter: auto !important;
```

Comment:
This is for Brave browser (mine) and probably some others, DaisyUI dialogs shows a scrollbar gutter it's a known bug ([issue link](https://github.com/saadeghi/daisyui/issues/3040)